jwplayer-appletv-web-app/js/utils/Promise.js

```diff
@@ -0,0 +1,235 @@
+/*! https://github.com/taylorhakes/promise-polyfill v6.0.2 by Taylor Hakes | MIT license */
```

Comment:
I've included licenses for `he.js` and `promise-polyfill.js` under the licenses directory. Are we using any other third-party projects for which we need to include a license?